ONSdigital/babbage
# remove conditional rendering logic for cookies banner

### What

Removed the conditional rendering logic for the cookies banner as this will now be handled client-side.

### How to review

- Pull `dp-frontend-renderer` and `sixteens` branches to test this locally - `feature/no-js-cookies-banner` and `feature/client-side-banner-rendering` respectively
- Check to see that banner is not present when `cookies_preferences_set=true` and that it is visible when no cookies have been set.
- Also check that the banner does not flash up and then disappear when first loading a page with cookies preferences set. I've tested this locally and with network throttling on via Google dev tools, and did not see any banner flashing.

### Who can review

Anyone but me
